corehq/apps/users/tasks.py

```diff
@@ -346,3 +346,29 @@ def process_reporting_metadata_staging():
     run_again = run_periodic_task_again(process_reporting_metadata_staging_schedule, start, duration)
     if run_again and UserReportingMetadataStaging.objects.exists():
         process_reporting_metadata_staging.delay()
+
+
+@task(queue='background_queue', acks_late=True)
+def loadtest_user_is_demo_user(commcare_user_id):
```

Comment:
With the test, I think this is a hard-to-parse name for this function and it has unexpected behaviour when used in context - I would have thought this was going to return a boolean (answering the question - "is this loadtest user a demo user?"). 
Maybe something like `apply_correct_demo_mode_to_loadtestuser` or something?